lib/libsh/libsimbol/libsimbol.sh

```diff
@@ -221,6 +222,71 @@ function core:log() {
     fi
 }
 #. }=-
+#. 1.7  Sanity Checks / Validation -={
+function validate_bash() {
+    local -i e=${CODE_FAILURE?}
+
+    local vv="/tmp/.simbol-bash-${BASH_VERSION}.verified"
```

Comment:
Good catch!

It's not amazing, but you've brought my attention to the fact that I did not make use of the abstraction variables.  I'll fix this and post a new CR for you.